app/models/concerns/filterable.rb

```diff
@@ -0,0 +1,13 @@
+module Filterable
+  extend ActiveSupport::Concern
+
+  module ClassMethods
```

Comment:
does this end up getting used anywhere? if not, please delete. if so, please include a doc comment that explains what it does and how it works.

Reply:
This Filterable module was intended to reduce repetitive code when applying multiple scopes in the controller. It dynamically applied filters based on provided parameters, but it's not currently in use, so I’m removing it to clean up the codebase


here is the full document:

https://www.justinweiss.com/articles/search-and-filter-rails-models-without-bloating-your-controller/

Reply:
The test is added now 

@aaronbrethorst